opencti-platform/opencti-front/src/private/components/settings/users/edition/UserEditionOverview.tsx

```diff
@@ -317,6 +325,27 @@ UserEditionOverviewComponentProps
             onFocus={handleChangeFocus}
             onChange={handleSubmitField}
           />
+          <div style={{ marginTop: 20 }}>
+            <Accordion expanded={openOptions} onChange={() => setOpenOptions(!openOptions)}>
```

Comment:
you do not need to control the expanded state of the Accordion if there is no additional thing to do on change.

```suggestion
            <Accordion>
```